tools/accuracy_checker/accuracy_checker/metrics/classification.py

```diff
@@ -345,7 +349,7 @@ def update(self, annotation, prediction):
         return 0
 
     def evaluate(self, annotations, predictions):
-        all_results = np.concatenate([t.squeeze() for t in self.results])
+        all_results = np.concatenate([t.squeeze() if len(t.shape) > 1 or t.shape[0] > 1 else t for t in self.results])
```

Comment:
1. ndim returns numer of dimensions
2. there is no guaranties that data is np arays
```suggestion
        all_results = np.concatenate([t.squeeze() if np.ndim(t) > 1 or np.shape(t)[0] > 1 else t for t in self.results])
```
Also possibly better to perform this modification during adding values to results instead

Reply:
Yes, this implementation of roc_auc_score metric was introduced for huge DLRM model in multibatch mode. Since huge DLRM is out of scope, it seems better to remove squeeze() of results at all.